config/routes.rb

```diff
@@ -298,6 +298,7 @@ def draw(routes_name)
 
       collection do
         get 'cards'
+        get ':project_folder_id/cards', to: 'projects#cards'
```

Comment:
the route will be like: 
`projects/1/cards` what reads like cards of project 1... 
I would add `folders` before `:project_folder_id` like `projects/folders/1/cards`